bin/verifyExports.js

```diff
@@ -0,0 +1,37 @@
+const index = require('../src/components/index')
+const findAllFilesInDirectories = require('./helpers/findAllFilesInDirectories')
+
+function verifyExports(files = nonspecJsFiles()) {
+  return files
+}
+
+export default verifyExports
+
+function nonspecJsFiles() {
+  const jsFilesWithSpec = findAllFilesInDirectories(['src/components'], '.js')
+  const jsFiles = [...jsFilesWithSpec].filter((file) => {
+    return !(
+      file.includes('spec') ||
+      file.includes('test') ||
+      file.includes('index')
+    )
+  })
+
+  const componentsFromFiles = jsFiles.map((file) =>
+    file
+      .split('/')
+      .slice(-1)[0]
+      .split('.')[0]
+      .toLowerCase()
```

Comment:
Should we lower case the components? Not sure if we'll ever have a `foobar` and a `FooBar` but this would make them be considered as the same thing

Reply:
Ah yeah and not only that we should be checking that exports from index are capitalized anyway. I almost accidentally exported a `faq` instead of a `Faq`. We could check this here